src/renderer/atlas/AtlasEngine.r.cpp

```diff
@@ -328,6 +333,9 @@ void AtlasEngine::_drawCursor()
     }
 
     _r.d2dRenderTarget->BeginDraw();
+    // We need to clip the area we draw in to ensure we don't
+    // accidentally draw into any neighboring texture atlas tiles.
+    _r.d2dRenderTarget->PushAxisAlignedClip(&clip, D2D1_ANTIALIAS_MODE_ALIASED);

```

Comment:
I don't actually understand why we need this clip! We are only filling or stroking within `0,0 .. cellSizeDIP` _already_; were we measurably "accidentally" drawing into the neighboring tiles? When? How?

Reply:
I'm a bit concerned that D2D might draw lines/rectangles outside of the coordinates I specify, be it due to off-by-fractional-1 bugs or due to AA. I think using a clip rect is more robust and doesn't hurt performance since we only seldomly draw a cursor.